tests/views/test_authentication.py

```diff
@@ -5,12 +5,41 @@
 from django.utils.encoding import force_text
 from django.utils.translation import ugettext_lazy as _
 
+from jwt import get_unverified_header
+
+from pytest import skip
+
 from rest_framework import status
 
 from rest_framework_jwt.authentication import JSONWebTokenAuthentication
 from rest_framework_jwt.compat import has_set_cookie_samesite
 from rest_framework_jwt.settings import api_settings
 
+from sys import version_info
+
+try:
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.hazmat.primitives.asymmetric import rsa
+
+    # DO NOT copy key_size=512 for any real code!
```

Comment:
Why?

Reply:
because it's probably to be considered insecure for any real world application, yet I wanted to keep the test case efficient.